Career/career.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <ul>
+        <img src="../Images/20220321113235643.jpg" alt="Reshitha" height="170px"></img>
+        <li><b>Name : Reshitha</b></li>
+        <li><b>Grade : 2nd Grade</b></li>
+    </ul></br>
+
+
+    <ul>
+        <li><img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img></li>
+        <li><b>Name : Skandhana</b></li>
+        <li><b>Grade : 1st Grade</b></li>
+    </ul>
+
+    <ul>
+        <li><img src="../Images/20220321113910381.jpg" alt="Kalyani" height="170px"></img></li>
+        <li><b>Name : Kalyani</b></li>
+        <li><b>Grade : Kinder Garden</b></li>
+    </ul>
```

Comment:
Have single list. Make each student an item in the list.

Reply:
done